src/renderer/store/handlers/messages.ts

```diff
@@ -210,10 +208,29 @@ export const updatePublicChannels = () => async (dispatch, getState) => {
         username: mainChannel.name
       })
     )
+    const importedChannels = electronStore.get('importedChannels') || {}
+    electronStore.set('importedChannels', {
```

Comment:
I would prefer to set this flag in redux rather than electronStore.
In redux some keys are persisted, so we should set this under a persisted key.

So instead of `electronStore.get` you would use selector. And instead of `electronStore.set` you would dispatch an action that reducer reacts to

Reply:
I just had to make setting general channel compatible because importedChannels from electronStore is already used in a few other places in the code, e.g in loadAllMessages.

Reply:
ahhhh,  I see, so this is not a new key you added, you're modyfing existing one?

Reply:
That's right. importedChannels exist and they were missing the general channel data - this was causing one of the problems with loading messages